yt_dlp/extractor/deuxm.py

```diff
@@ -0,0 +1,77 @@
+from .common import InfoExtractor
+from ..utils import url_or_none
+
+
+class DeuxMIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?2m\.ma/[^/]+/replay/single/(?P<id>(?:[A-Za-z0-9_.]{1,24})+)'
```

Comment:
What do u intend by `(?:[A-Za-z0-9_.]{1,24})+`? Did you mean just `[\w.]{1,24}`?

Reply:
indeed, i do not have much knowledge of regex